cloud_experts_tutorials/cloud_experts_getting_started/cloud-experts-getting-started-setup.adoc

```diff
@@ -0,0 +1,227 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-getting-started-setup"]
+= Tutorial: Setup
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-getting-started-setup
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2023-11-13
+
+There are currently two supported credential methods when creating a {product-title} (ROSA) cluster. One method uses an IAM user with the `AdministratorAccess` policy. The second and *recommended* method uses Amazon Web Services (AWS) Security Token Service (STS). 
+//To be added when the ROSA with STS Explained tutorial is published:
+//For more information, see the xref../cloud_experts_tutorials/cloud_experts_rosa_with_sts_explained.adoc#id[ROSA with STS Explained] tutorial. This workshop uses the STS method.
+
+== Prerequisites
+
+Please review the prerequisites listed in the xref:../../rosa_planning/rosa-cloud-expert-prereq-checklist.adoc#rosa-cloud-expert-prereq-checklist[Prerequisites for ROSA with STS] checklist.
+
+You will need the following information from your AWS account:
+
+* AWS IAM user
+* AWS access key ID
+* AWS secret access key
+
+== Setting up a Red Hat account
+. If you do not have a Red Hat account, create one on the link:https://console.redhat.com/[Red Hat console].
+. Accept the required terms and conditions.
+. Then check your email for a verification link.
+
+== Installing the AWS CLI
+* Install the link:https://aws.amazon.com/cli/[AWS CLI] for your operating system.
+
+== Enabling ROSA
+
+[NOTE]
+====
+Only complete this step if you have *not* enabled ROSA in your AWS account.
+====
+
+. Visit the link:https://console.aws.amazon.com/rosa[AWS console] to enable your account to use ROSA.
+. Click on the orange *Enable OpenShift* button.
++
+image::cloud-experts-getting-started-setup-enable.png[]
+
+. After about a minute, a green *service enabled* bar should appear.
++
+image::cloud-experts-getting-started-setup-enabled.png[]
+
+== Installing the ROSA CLI
+. Install the link:https://console.redhat.com/openshift/downloads[ROSA CLI] for your operating system.
+. Download and extract the relevant file for your operating system by using the following command:
++
+[source,terminal]
+----
+tar -xvf rosa-linux.tar.gz
+----
+. Save the file to a location within your path by using the following command:
```

Comment:
"your path" sounds weird. Maybe "Save the file to a location with your computer by using the following command."?